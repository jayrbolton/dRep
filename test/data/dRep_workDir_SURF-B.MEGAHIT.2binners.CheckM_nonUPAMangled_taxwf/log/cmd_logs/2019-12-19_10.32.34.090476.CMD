prodigal -i /home/sumin/kbsdk-workspace/dRep/test/data/binned_contig_files_3bins/Bin.008.fasta -d /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.008.fasta.fna -a /home/sumin/kbsdk-workspace/dRep/test/data/res.dRep.txwf.uniq/data/prodigal/Bin.008.fasta.faa -m -p meta
