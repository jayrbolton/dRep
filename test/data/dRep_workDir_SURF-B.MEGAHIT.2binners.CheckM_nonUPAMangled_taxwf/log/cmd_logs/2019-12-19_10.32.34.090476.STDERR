-------------------------------------
PRODIGAL v2.6.3 [February, 2016]         
Univ of Tenn / Oak Ridge National Lab
Doug Hyatt, Loren Hauser, et al.     
-------------------------------------
Request:  Metagenomic, Phase:  Training
Initializing training files...done!
-------------------------------------
Request:  Metagenomic, Phase:  Gene Finding
Finding genes in sequence #1 (51408 bp)...done!
Finding genes in sequence #2 (46144 bp)...done!
Finding genes in sequence #3 (168360 bp)...done!
Finding genes in sequence #4 (36840 bp)...done!
Finding genes in sequence #5 (98122 bp)...done!
Finding genes in sequence #6 (38403 bp)...done!
Finding genes in sequence #7 (5550 bp)...done!
Finding genes in sequence #8 (3603 bp)...done!
Finding genes in sequence #9 (50923 bp)...done!
Finding genes in sequence #10 (42893 bp)...done!
Finding genes in sequence #11 (51754 bp)...done!
Finding genes in sequence #12 (7556 bp)...done!
Finding genes in sequence #13 (6133 bp)...done!
Finding genes in sequence #14 (57096 bp)...done!
Finding genes in sequence #15 (48728 bp)...done!
Finding genes in sequence #16 (19523 bp)...done!
Finding genes in sequence #17 (73065 bp)...done!
Finding genes in sequence #18 (32190 bp)...done!
Finding genes in sequence #19 (8088 bp)...done!
Finding genes in sequence #20 (52268 bp)...done!
Finding genes in sequence #21 (107486 bp)...done!
Finding genes in sequence #22 (13522 bp)...done!
Finding genes in sequence #23 (2481 bp)...done!
Finding genes in sequence #24 (2076 bp)...done!
Finding genes in sequence #25 (10843 bp)...done!
Finding genes in sequence #26 (35480 bp)...done!
Finding genes in sequence #27 (34029 bp)...done!
Finding genes in sequence #28 (49332 bp)...done!
Finding genes in sequence #29 (24519 bp)...done!
Finding genes in sequence #30 (40668 bp)...done!
Finding genes in sequence #31 (6447 bp)...done!
Finding genes in sequence #32 (20419 bp)...done!
Finding genes in sequence #33 (103403 bp)...done!
Finding genes in sequence #34 (37767 bp)...done!
Finding genes in sequence #35 (28353 bp)...done!
Finding genes in sequence #36 (30012 bp)...done!
Finding genes in sequence #37 (26830 bp)...done!
Finding genes in sequence #38 (45608 bp)...done!
Finding genes in sequence #39 (39847 bp)...done!
Finding genes in sequence #40 (8995 bp)...done!
Finding genes in sequence #41 (20377 bp)...done!
Finding genes in sequence #42 (37439 bp)...done!
Finding genes in sequence #43 (8666 bp)...done!
Finding genes in sequence #44 (13715 bp)...done!
Finding genes in sequence #45 (23179 bp)...done!
Finding genes in sequence #46 (3598 bp)...done!
Finding genes in sequence #47 (44502 bp)...done!
Finding genes in sequence #48 (50672 bp)...done!
Finding genes in sequence #49 (51876 bp)...done!
Finding genes in sequence #50 (2451 bp)...done!
Finding genes in sequence #51 (9729 bp)...done!
Finding genes in sequence #52 (8373 bp)...done!
Finding genes in sequence #53 (147991 bp)...done!
Finding genes in sequence #54 (35801 bp)...done!
Finding genes in sequence #55 (49751 bp)...done!
Finding genes in sequence #56 (13801 bp)...done!
Finding genes in sequence #57 (25870 bp)...done!
Finding genes in sequence #58 (4532 bp)...done!
Finding genes in sequence #59 (116755 bp)...done!
Finding genes in sequence #60 (34423 bp)...done!
Finding genes in sequence #61 (49590 bp)...done!
Finding genes in sequence #62 (45394 bp)...done!
Finding genes in sequence #63 (31834 bp)...done!
Finding genes in sequence #64 (17587 bp)...done!
Finding genes in sequence #65 (2884 bp)...done!
Finding genes in sequence #66 (41518 bp)...done!
Finding genes in sequence #67 (53757 bp)...done!
Finding genes in sequence #68 (2026 bp)...done!
Finding genes in sequence #69 (16975 bp)...done!
Finding genes in sequence #70 (6388 bp)...done!
Finding genes in sequence #71 (14387 bp)...done!
Finding genes in sequence #72 (40997 bp)...done!
Finding genes in sequence #73 (55539 bp)...done!
Finding genes in sequence #74 (5101 bp)...done!
Finding genes in sequence #75 (14952 bp)...done!
Finding genes in sequence #76 (8882 bp)...done!
Finding genes in sequence #77 (76030 bp)...done!
Finding genes in sequence #78 (5548 bp)...done!
Finding genes in sequence #79 (88713 bp)...done!
Finding genes in sequence #80 (6022 bp)...done!
Finding genes in sequence #81 (182874 bp)...done!
Finding genes in sequence #82 (52818 bp)...done!
Finding genes in sequence #83 (89519 bp)...done!
Finding genes in sequence #84 (2297 bp)...done!
Finding genes in sequence #85 (40635 bp)...done!
Finding genes in sequence #86 (2313 bp)...done!
Finding genes in sequence #87 (7817 bp)...done!
Finding genes in sequence #88 (39731 bp)...done!
Finding genes in sequence #89 (21439 bp)...done!
Finding genes in sequence #90 (3994 bp)...done!
Finding genes in sequence #91 (142821 bp)...done!
Finding genes in sequence #92 (2975 bp)...done!
Finding genes in sequence #93 (65965 bp)...done!
Finding genes in sequence #94 (42390 bp)...done!
Finding genes in sequence #95 (42971 bp)...done!
Finding genes in sequence #96 (29265 bp)...done!
Finding genes in sequence #97 (91633 bp)...done!
Finding genes in sequence #98 (8330 bp)...done!
Finding genes in sequence #99 (4192 bp)...done!
Finding genes in sequence #100 (60654 bp)...done!
Finding genes in sequence #101 (67662 bp)...done!
Finding genes in sequence #102 (9955 bp)...done!
Finding genes in sequence #103 (9777 bp)...done!
Finding genes in sequence #104 (30644 bp)...done!
